src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write code here
+const lilist = document.querySelectorAll('li');
+
+lilist.forEach(li => {
+  const span = document.createElement('span');
+  const firstChild = li.firstChild;
+
+  span.textContent = firstChild.textContent;
+  li.insertBefore(span, firstChild);
+  firstChild.textContent = '';
+});
+
+const tree = document.querySelector('.tree');
```

Comment:
please move this declaration to the top of the file, right before `document.querySelectorAll('li');`